lms/static/js/verify_student/views/step_view.js

```diff
@@ -101,4 +103,4 @@
 
     });
 
- })( jQuery, _, _.str, Backbone, gettext );
+ })( jQuery, _, Backbone, gettext );
```

Comment:
The issue is that this code is NOT using requirejs, but it is called by test code that IS using requirejs. There's extensive shimming going on here when the tests run (see above in spec/main.js).